includes/discovery/sensors/count/dhcpatriot.inc.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/*
+ *
+ * OIDs obtained from First Network Group Inc. DHCPatriot operations manual version 6.4.x
+ * Found here: http://www.network1.net/products/dhcpatriot/documentation/PDFs/v64xmanual-rev1.pdf
+ *
+*/
+
+$oids = array(
+    1 => array(
+        'type'  => 'dhcpatriotDatabaseThreads',
+        'descr' => 'Database Threads',
+        'oid'   => '.1.3.6.1.4.1.2021.50.45'
+    ),
+    2 => array(
+        'type'  => 'dhcpatriotDatabaseQueriesPerSecond',
+        'descr' => 'Database Queries Per Second',
+        'oid'   => '.1.3.6.1.4.1.2021.50.46'
+    ),
+    3 => array(
+        'type'  => 'dhcpatriotDHCPv4LeasesPerSecond',
+        'descr' => 'DHCPv4 Leases Per Second',
+        'oid'   => '.1.3.6.1.4.1.2021.50.70'
+    ),
+    4 => array(
+        'type'  => 'dhcpatriotDHCPv6LeasesPerSecond',
+        'descr' => 'DHCPv6 Leases Per Second',
+        'oid'   => '.1.3.6.1.5.1.2021.50.140'
+    ),
+    5 => array(
+        'type'  => 'dhcpatriotLicenseExpiration',
+        'descr' => 'License Expiration Days Remaining',
+        'oid'   => '.1.3.6.1.4.1.2021.51.12.4.1.2.7.76.73.67.69.78.83.69.1'
+    )
+);
+
+$class = 'count';
+$divisor = 1;
+$multiplier = 1;
+$low_limit = null;
+$low_warn_limit = null;
+$warn_limit = null;
+$high_limit = null;
+$poller_type = 'snmp';
+$entPhysicalIndex = null;
+$entPhysicalIndex_measured = null;
+$user_func = null;
+$group = null;
+
+$current_time = time();
+
+foreach ($oids as $index => $entry) {
+    $oid = $entry['oid'];
+    $type = $entry['type'];
+    $descr = $entry['descr'];
+
+    $current = snmp_get($device, $oid, '-Oqv');
```

Comment:
Could just use snmp_get_multi_oid() to fetch all these at once instead of one at a time.

Reply:
I'll have to see what I can do to rework these. Your suggestion might reduce the poller time too if I change the poller files as well.

<img width="1321" alt="Screen Shot 2020-06-09 at 10 14 00 AM" src="https://user-images.githubusercontent.com/43786590/84166173-2a234100-aa3a-11ea-9e5b-367be2637676.png">